openlibrary/templates/type/list/view.html

```diff
@@ -206,6 +206,8 @@ <h1>$list.name</h1>
                         <h3 class="booktitle">
                             <a href="$seed.url" class="results">$seed.title</a>
                         </h3>
+                        $if seed.type in ['edition', 'work'] and doc.subtitle:
+                            <h4 class="work-subtitle">$doc.subtitle</h4>
```

Comment:
Am not 100% convinced h4 is appropriate is here, but also not a blocker.

Reply:
@bpmcneilly may have comments about the document structure. Given the title is currently a h3, not opposed to merging as is. Seems to add useful data to the document / display. Happy for there to be a followup issue opened if a different semantic structure is useful.